apps/blog/src/api/query-helpers/user.ts

```
@@ -0,0 +1,34 @@
+      id: true,
+    } satisfies Prisma.SessionSelect;
+  }
+
+  // public static activeSessionWhere() {
+  //   return {
+  //     expiresAt: {
+  //       gt: new Date(),
+  //     },
+  //   } satisfies Prisma.SessionWhereInput;
+  // }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)